### PR TITLE
remove default admin url suffix

### PR DESCRIPTION
**Pull request by wesleymccollam**

### CODE REVIEW

File: internal/types/definitions.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 // Configuration used by the provider and resources
 type ProviderConfiguration struct {
 	HttpsHost      string
+	AdminApiPath   string
```
Comment (henryrecker-pingidentity) [inline internal/types/definitions.go:11]: I don't think we need this in this struct if it's never used in any resource

File: internal/provider/provider.go [hunk 2]
```diff
@@ -312,7 +337,7 @@ func (p *pingfederateProvider) Configure(ctx context.Context, req provider.Confi
 	clientConfig.DefaultHeader["X-BypassExternalValidation"] = strconv.FormatBool(xBypassExternalValidation)
 	clientConfig.Servers = client.ServerConfigurations{
 		{
-			URL: httpsHost + "/pf-admin-api/v1",
+			URL: httpsHost + adminApiPath,
```
Comment (henryrecker-pingidentity) [inline internal/provider/provider.go:340]: Do we need to have these as separate variables? Would it make more sense to just have one entire host variable including the admin api path?

**Review by henryrecker-pingidentity — APPROVED** [review]

File: internal/provider/provider.go (hunk 2 above)
Comment (patrickcping) [reply internal/provider/provider.go]: This is a breaking change for current users - I've asked Wes to revert

Comment (wesleymccollam) [reply internal/provider/provider.go]: fixed and merged https://github.com/pingidentity/terraform-provider-pingfederate/pull/170

**Pull request merged**